### ISSUE TITLE
--copy-gds fails with optional files if the destination directory doesn't exists

### ISSUE THREAD

**Issue by mbalestrini** [mattvenn/multi_project_tools#38]
I got an error on this line when it was trying to copy a project `spef` file because the destination `spef` folder didnt exist:

https://github.com/mattvenn/multi_project_tools/blob/0f7c5722fc47afad85f750f5e1e44256c9e6bb45/collect.py#L126

**Comment by mattvenn** [comment]
and you don't have a spef folder in caravel_user_project?
but yes, should fail more gracefully probbably

**Comment by mbalestrini** [comment]
no, I had no spef folder. At least not before running `make user_project_wrapper`
